packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -304,6 +304,18 @@ mixin WidgetsBinding on BindingBase, SchedulerBinding, GestureBinding, RendererB
       );
     }
 
+    // Expose the ability to send Widget rebuilds as [Timeline] events.
```

Comment:
Nit: place these with the extensions above.

Reply:
Done.